File: python/hello.py
from wsgiref.simple_server import make_server

def app(environ, start_response):
    start_response('200 OK', [('Content-Type', 'text/html')])
    return ['<p>Hello, WSGI!</p>']

httpd = make_server('', 8070, app)
httpd.serve_forever()

File: ruby/hello.rb
require 'rack'

app = proc do |env|
  ['200', {'Content-Type' => 'text/html'}, ['<p>Hello, Rack!</p>']]
end

Rack::Handler::WEBrick.run(app, :Port => 8090, :Host => '0.0.0.0')
